src/styles/generic/_code.scss

```diff
@@ -80,16 +80,21 @@ pre code {
 
 // It's not uncommon to have code in markdown headings.
 // In these cases we should remove the border.
+// There is also a utility class, .unstyled-code, that can be used
+// if we're using <code> blocks in new contexts.
+// For example: <span class="unstyled-code">{{ description | md | safe }}</span>
 h1 code,
 h2 code,
 h3 code,
 h4 code,
 h5 code,
-h6 code {
+h6 code,
+.unstyled-code code {
   background: transparent;
   border: 0;
   color: inherit;
   font-size: inherit;
+  font-style: inherit;
```

Comment:
at what point is this just `font: inherit`?

Reply:
I think the difference is we don't want it to use the same font face (we want to use the mono one). I'm not sure about the specificity with shorthands but I'm happy to try `font: inherit` to see if it still gives us the result we want.

Reply:
Follow up: I tried `font: inherit` but that made it stop using the mono typeface